usr/share/rear/skel/default/etc/scripts/system-setup

```diff
@@ -151,15 +152,17 @@ else
     rear_debug_options=''
 fi
 
-# Launch rear recover automatically:
+reboot_command="${RECOVERY_REBOOT_COMMAND:-reboot}"
+
+# Launch "rear recover" automatically:
 if automatic_recovery ; then
     choices=( "View Relax-and-Recover log file(s)"
               "Go to Relax-and-Recover shell"
             )
     echo -e "\nLaunching 'rear recover' automatically\n"
     if rear $rear_debug_options recover ; then
         echo -e "\n'rear recover' finished successfully\n"
-        choices+=( "Reboot" )
+        choices+=( "$reboot_command" )
```

Comment:
And then you can simply call it here without the extra `reboot_command` variable...

Reply:
yes